### ISSUE TITLE
Email templates need work

### ISSUE THREAD

**Issue by fkleedorfer** [researchstudio-sat/webofneeds#2685]
* Don't HTML encode text when sending plain text mail
* forwarded messages should be described as such (message from X via Y) 
* reply needs have no title but we should generate an explanation. '(no title)' looks bad

**Comment by quasarchimaere** [comment]
I think this would be an "epic"-issue,

What should be done first is:
- Implement a way to parse need-content (e.g. details) serverSide
- Implement a way to parse message-content (e.g. details/messageType) serverSide

Once we have done the prelim. items we can then parse email message content way easier, and also add a sort-of framework solution to make java-side implementation easier (e.g. bots)

**Comment by fkleedorfer** [comment]
I agree that we can spend much more time on mail notifications. This ticket is about problems that currently occur and need urgent fixing.
